2021/30xxx/CVE-2021-35093.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Expected location (2021/35xxx/CVE-2021-35093.json) of file does not match actual location (2021/30xxx/CVE-2021-35093.json)